solutions/usaco-715.mdx

```diff
@@ -98,4 +98,30 @@ public class MaxCross {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+with open('maxcross.in') as read:
+	n, k, b = map(int, read.readline().split())
+	id, left, right, tmp = [0] * (n + 1), 1, k, 0
```

Comment:
don't use `id` as a var name since it overrides some builtin function

Reply:
okay